Tests/StoreKitUnitTests/TestHelpers/StoreKitTestHelpers.swift

```diff
@@ -32,8 +32,7 @@ extension StoreKitConfigTestCase {
     func createTransactionWithPurchase() async throws -> Transaction {
         let product = try await self.simulateAnyPurchase()
 
-        let latestTransaction = await product.latestTransaction
-        let transaction = try XCTUnwrap(latestTransaction)
+        let transaction = try await XCTAsyncUnwrap(await product.latestTransaction)
```

Comment:
Unfortunately both `await`s are required because of how `@autoclosure`s work.

Reply:
cool (cool)